src/components/RJST/components/Filters/index.tsx

```diff
@@ -62,7 +62,7 @@ export interface FiltersProps {
 
 // TODO remove when we have implemented a dark theme
 const darkStyles: SxProps = {
-	color: color.text.value,
+	color: token('color.text'),
 	backgroundColor: 'white',
```

Comment:
Do we want to replace this `white` with a token as well?

Reply:
See comment above